VERSION

```diff
@@ -1 +1 @@
-0.1.2
+1.0.0
```

Comment:
@textpipe/developers any opinions on this? we're breaking backwards compatibility, so I guess this should indeed be bumped?

Reply:
I don't think we need to break backward compatibility for this. We can just accept either a tuple or a string as an element in a pipeline.